voyager-publish/index.js

```diff
@@ -10,21 +10,47 @@ export const publish = async ({
   logger = console
 }) => {
   // Fetch measurements
-  const { rows: measurements } = await pgPool.query(`
-    SELECT
-      id,
-      zinnia_version,
-      participant_address,
-      status_code,
-      end_at,
-      inet_group,
-      car_too_large,
-      cid
-    FROM measurements
-    LIMIT $1
-  `, [
-    maxMeasurements
-  ])
+  let measurements
+  {
+    const pgClient = await pgPool.connect()
+    try {
+      await pgClient.query('BEGIN ISOLATION LEVEL SERIALIZABLE')
+
+      // Delete published measurements
```

Comment:
This comment does not describe the SQL statement. Could you please update or remove it?

Reply:
> This comment does not describe the SQL statement. Could you please update or remove it?

@juliangruber this is not addressed yet, PTAL

Reply:
Why not? Isn't that exactly what we're doing: Deleting measurements that we just published?

Reply:
ooh I was looking at the wrong comment in line 94 :D Sorry about that